### DISCUSSION TITLE
Missing easy instructions how to stream to server

### DISCUSSION THREAD

**Discussion by borgqueenx** [bluenviron/mediamtx#2109] [Questions]
I have a wyze bridge, where i can put optional instructions a RTSP stream to cast to. this is the option, as given from the wyze bridge documentation.
![qpq5eDR](https://github.com/bluenviron/mediamtx/assets/20416925/2e58df67-1895-4601-942c-34df92c4dc5f)

What I am missing is the easy instructions how to make this land on my externally hosted server, where i have mediamtx installed, and then cast the stream to a webpage so it can be reached from http://ipaddress/mystream

In the Mediamtx documentation, I am reading about publishing. but it speaks about "file.ts". It does not exist on the server. I suppose this is the media file that it will stream, but i cannot find the way how to create this media file, how it can land on the server and how it communicates with my local raspberry pi that tries to cast a camera to a RTSP stream.

Perhaps including a example in the documenation would also be helpful..

help would be appriciated!

**Comment by aler9** [comment]
Hello, if you want to forward a live stream from Wyze Bridge to an instance of MediaMTX, you can put into `LIVESTREAM` the URL described in the [RTMP clients](https://github.com/bluenviron/mediamtx#rtmp-clients) section of the README:

```yml
- CAM_NAME: Baby Cam
  Quality: HD70
  LIVESTREAM: rtmp://mediamtx-ip/mystream
```

You stream will be automatically available in a web page that uses WebRTC:

```
http://mediamtx-ip:8889/mystream
```

or HLS:


```
http://mediamtx-ip:8888/mystream
```

[a comment by an automated account is not shown]
